test/fixtures/mson/enum-sample.json

```diff
@@ -153,6 +166,19 @@
                       "value": {
                         "element": "enum",
                         "attributes": {
+                          "enumerations": {
+                            "element": "array",
+                            "content": [
+                              {
+                                "element": "string",
+                                "content": "3"
+                              },
+                              {
+                                "element": "string",
+                                "content": "4"
+                              }
+                            ]
```

Comment:
This matches any `string`. Are you missing `fixed` attribute on the `string` elements above?